nucypher/cli/literature.py

```diff
@@ -423,7 +423,7 @@
 
 SUCCESSFUL_RETARGET = "Successfully re-targeted {contract_name} proxy to {target_address}"
 
-SUCCESSFUL_RETARGET_TX_BUILT = "Transaction to retarget {contract_name} proxy to {target_address} was built:"
+SUCCESSFUL_RETARGET_TX_BUILT = "Successfully built transaction to retarget {contract_name} proxy to {target_address}:"
```

Comment:
It's almost worth documenting the thought process and tone used throughout here.

Reply:
Haha really it was just from the top of the file to the bottom i.e. the style used higher up in the file, was replicated lower down in the file.